src/messages/ru/hipanel:stock.php

```diff
@@ -157,4 +157,7 @@
     'Edit properties' => 'Редактировать свойства',
     'Last move with descr' => 'Последнее перемещение с опис.',
     'Administrative' => 'Административный',
+    'Fast move' => 'Быстрое перемещение',
+    'Type server names using a space, comma or on a new line' => 'Введите имена серверов через пробел, запятую или с новой строки.',
```

Comment:
```suggestion
    'Type server names, delimited with a space, comma or a new line' => 'Введите имена серверов через пробел, запятую или с новой строки.',
```

Ато полчается "Введите имена серверов пробелом, запятой ...."